Description: Increase epsilon in a DeterminantalRepresentations that
 was failing on arm64.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1680
Origin: https://github.com/Macaulay2/M2/pull/1682
Last-Update: 2020-12-12

--- a/M2/Macaulay2/packages/DeterminantalRepresentations.m2
+++ b/M2/Macaulay2/packages/DeterminantalRepresentations.m2
@@ -1352,7 +1352,7 @@
 reps = detRep fCC
 assert(all(reps, L -> clean(1e-9, fCC - det L) == 0))
 reps = detRep(fCC, HyperbolicPt => matrix{{3_RR},{2},{-7}}) -- representations are not monic
-assert(all(reps, L -> clean(1e-6, fCC - det L) == 0))
+assert(all(reps, L -> clean(1e-5, fCC - det L) == 0))
 ///
 
 TEST /// -- Cubic case: homogeneous, 3 variables
